build-logic/settings.gradle.kts

```diff
@@ -71,6 +71,9 @@ include("cleanup")
 include("idea")
 include("lifecycle")
 
+// Third party installation build logic for root project - please preserve alphabetical order
+include("third-party-installations")
```

Comment:
Would it make sense to move this plugin to the `performance-testing` project instead of adding a new subproject here?